install.rb

```diff
@@ -260,15 +260,12 @@ def prepare_installation
     $operatingsystem = Facter.value :operatingsystem
   end
 
+  # populates constants used to specify default Windows directories
+  require 'win32/dir' if $operatingsystem == "windows"
```

Comment:
Yeah, it probably indicates a failure of some kind to setup an environment / package it.